docs/system_overview/data_objects.md

```diff
@@ -264,30 +264,50 @@ Trim refers to unlinking one or more replicas mapped to an existing data object.
 
 ![Trim](../images/trim_01.drawio.png)
 
-Trim requires 2 inputs:
+Trim requires 1 input:
 
 1. An existing data object with at least two replicas which are at rest and can be unlinked.
-2. A minimum number of good replicas to retain.
 
-Trim selects stale replicas first, then the good replicas are trimmed from oldest to youngest, down to the specified minimum number of good replicas.
+Trim requires a minimum number of replicas to retain at completion. The default minimum number of replicas to retain is 2.
 
-If the data object has fewer than the minimum number of good replicas, trim is not *allowed* in any case.
+Trim selects stale replicas first - trimming from oldest to youngest - until the specified minimum number of replicas is reached. If more replicas can be trimmed, then the good replicas are trimmed from oldest to youngest, down to the specified minimum number of replicas. The specified number of replicas cannot be less than 1. In other words, trim cannot completely remove a data object.
 
-In the following cases, ‘X’ represents a stale replica, ‘&’ represents a good replica, and ‘-’ indicates a replica which does not exist. This table shows the result of a data object targeted for a trim with a minimum good replica count of 1. The table shows the starting and ending statuses of 2 replicas for a single data object as a string such that the status of replica 0 is the first character and the status of replica 1 is the second character.
+If the data object has fewer than the minimum number of replicas, trim is not *allowed* in any case.
+
+In the following cases, ‘X’ represents a stale replica, ‘&’ represents a good replica, and ‘-’ indicates a replica which does not exist. This table shows the result of a data object targeted for a trim with a minimum replica count of 1. The table shows the starting and ending statuses of 2 replicas for a single data object as a string such that the status of replica 0 is the first character and the status of replica 1 is the second character.
```

Comment:
i don't think the last sentence is correct for this next table... there is no string of multiple characters.

Reply:
Hmm... I see what you mean. How's this?
```suggestion
In the following cases, ‘X’ represents a stale replica, ‘&’ represents a good replica, and ‘-’ indicates a replica which does not exist. This table shows the result of a trim targeting a data object with 2 replicas and a minimum replica count of 1.
```

Reply:
yep - seems to cover it.